### ISSUE TITLE
Did you implement generation of RGD images?

### ISSUE THREAD

**Issue by jlian2** [singh-hrituraj/PixelCNN-Pytorch#4]
I think generation code should be different: meaning, no softmax and we dont need to generate pixel by pixel?
